modules/bot_channel.py

```diff
@@ -5,5 +5,9 @@
 class Module(ModuleManager.BaseModule):
     @utils.hook("received.numeric.001")
     def do_join(self, event):
-        event["server"].send_join(event["server"].get_setting("bot-channel",
-            "#bitbot"))
+        try:
```

Comment:
I'd much rather this not be done with a try/except, something like:

```event["server"].send_join(event["server"].get_setting("bot-channel", self.bot.config.get("bot-channel", "#bitbot")))```

That way it'll first try the server's `bot-channel`, then try the config's `bot-channel` then fall back to `#bitbot`

Reply:
That said that's going to need a change here https://github.com/jesopo/bitbot/blob/d1bbd8cf/src/Config.py#L14 that removes keys that have empty values. if you do the above, I'll make that change and merge this in.

Reply:
I've done the above mentioned change to `Config.py`